src/language/dafnyInstallation.ts

```diff
@@ -54,7 +70,7 @@ function getDafnyPlatformSuffix(): string {
 
 function getDafnyDownloadAddress(): string {
   const baseUri = LanguageServerConstants.DownloadBaseUri;
-  const version = LanguageServerConstants.RequiredVersion;
+  const version = getConfiguredVersion();
```

Comment:
I would have decoupled the setting from this file and passed the version to the constructor of this class instead, but it's a stylistic issue.

Reply:
Most of the version handling is rather self-contained by now, i.e., the only version that is present outside this module is the one received during startup. The configured and installed version is only present and used within this module. Therefore, I think we can leave it as-is for the moment.
However, I've got the feeling that the class/module got a bit messy. I'll look if there are some reasonable refactorings.

Last but not least, I want to give automated integration testing another try. There were some code changes that were easy to miss. The following tests would be desirable here:

- `latest` installs and launches the version specified by `LatestVersion`
- Any of the *supported* versions can be installed and launched